test/benchee/benchmark/measure/memory_test.exs

```diff
@@ -37,7 +37,7 @@ defmodule Benchee.MemoryMeasureTest do
       # capture it.
       capture_io(fn ->
         Memory.measure(fn -> exit(:kill) end)
-        Process.sleep(1)
+        Process.sleep(10)
```

Comment:
?

Reply:
I think this will fix the flakiness we've been seeing in this test. Basically, when it's flaky it's because a process that we expected to exit hadn't exited yet. If we wait a little bit longer for the process that's generating the error (caused by the `exit(:kill)` call) to actually exit, then we shouldn't see the flakiness.